assets/queries/cloudFormation/ec2_network_acl_protocols/query.rego

```diff
@@ -33,3 +33,23 @@ checkValue(protocol) {
 	is_number(protocol)
 	protocol == 58
 }
+
+checkValue(protocol) {
```

Comment:
Couldn't replace `is_number` to `to_number`, since it returns `false` when can't convert, instead of duplicating code?

Reply:
Corrected per your suggestion